apps/docs/src/app/platform/component-docs/platform-forms/platform-form-generator/platform-form-generator-examples/platform-form-generator-programatic-submit.component.ts

```diff
@@ -0,0 +1,187 @@
+import { Component, ViewChild } from '@angular/core';
+import { Validators } from '@angular/forms';
+import { of } from 'rxjs';
+import { delay } from 'rxjs/operators';
+
+import { FdDate } from '@fundamental-ngx/core';
+import { DynamicFormItem, FormGeneratorComponent } from '@fundamental-ngx/platform';
```

Comment:
same

Reply:
Platform does not support deep imports yet